### ISSUE TITLE
C3: feed Chapter 5's memberships to a fitted consequent and measure held-out — the one arrow that is missing

### ISSUE THREAD

**Issue by fundthmcalculus** [fundthmcalculus/grad-school#204]
Closing **C3**, the item Chapter 7's tracker still lists as *not started* and Chapter 8 calls "the most interesting work remaining".

## What is actually missing

Not a model family, and not a dataset. **One arrow.**

Chapter 5 already produces a graded membership matrix and then throws it away. `reproduce/tables/table_5_4_ch5_g1_scaling.py:92`:

```python
U = np.vstack([MS.block_membership(b, Dstar) for b in blocks])
```

`U` is normalized to a partition of unity and then used only to compute its own residual, `np.abs(s[covered] - 1.0)`. Every other Chapter 5 path calls `MS.assign` — a hard `argmin` over minimax distance — and scores that with `adjusted_rand_score`.

So the chapter generates FIS antecedents and never fits a model on them. Verified: `gated-minimax-selection/` computes **no supervised metric at all** (all eleven `from sklearn.metrics import` lines import `adjusted_rand_score`), and `git log --all --pickaxe-regex` over 1,424 commits shows it never did.

## The bridge already exists

`tribblefis.regression.solve_tsk_consequents_from_firing(firing_strengths_train, ...)` takes a **precomputed firing matrix**. That is exactly the interface this needs — `U` goes in as `firing_strengths_train`, consequents come out, `predict_tsk` scores held-out. No new machinery.

## Scope

Add `reproduce/experiments/ch5_end_to_end.py` (a generator under `reproduce/tables/` if it earns a table):

1. Build `Dstar` → minimax transform → block selection → `MS.block_membership` → `normalize_partition`, giving `U` of shape `(blocks, n)`.
2. Split by point index. Slice `U` into train/test columns — **do not re-run selection per fold**; selection is unsupervised and re-running it leaks nothing but wastes time and changes the block set between folds.
3. `solve_tsk_consequents_from_firing(U_train.T, ...)` → `predict_tsk` on the held-out columns.
4. Report a **supervised** score — accuracy/F1 for classification, R² for regression — beside the Chapter 4 arm on identical splits, at the ten-seed protocol.

### Datasets

**bodyfat first.** Both arms already run on the same 252 rows: `FuzzySystemsExperiments/bodyfat_ivat.py` for the iVAT side, `FuzzySystemsExperiments/bodyfat.py` for `TribbleRegressor` (R² = 0.647, 5×5-fold). It has coordinates, so both constructions can run — which is precisely what §5.4 demands, *"compared against the Chapter 4 Gaussian construction on data where both can run"*.

**ECG5000 second**, and it is the one that matters more: `reproduce/tables/table_3_7_g2_downstream.py` already builds the DTW `Dstar` and the blocks and stops at `adjusted_rand_score` (lines 285/297/312/319). It is genuinely non-coordinate, so it exercises the regime Chapter 5's premise rests on.

### Not phishing

PhiUSIIL cannot be a venue for this. Chapter 5 needs a dense `N × N` dissimilarity matrix and PhiUSIIL is 235,795 rows — **0.4 TB at float64, 0.2 TB at float32**. Against bodyfat's 508 KB and ECG5000's 200 MB. The existing PhiUSIIL FIS results (`experiments/fis-to-neural-net/outputs/phiusiil.md`, test error 0.0060; Table 4.1's `acc = 0.997 ± 0.001`) are Chapter 4 MoG results and are the **baseline arm**, not the deliverable.

A fuzzy tree on phishing also already exists — `tribble-fis/tribble-tree/demo_phishing.py` fits `FuzzyClassificationTree` and prints accuracy. Its docstring opens *"End-to-end demo"*, which is where the ambiguity in C3 came from. It does not close C3, because its antecedents come from `gauss_math.create_gaussian_membership_dict`.

## Expect a negative, and report it anyway

`gated-minimax-selection/notes/phase6_soft_validation.py` already probed the graded signal and found the normalized minimax membership is **a constant step per cluster with no boundary resolution**, scoring worse than crisp 0/1 labels against the analytic posterior (Brier 0.136 / 0.200 / 0.208 / 0.122 fuzzy against 0.096 / 0.042 / 0.016 / 0.000 hard — `notes/MF_PROGRESS_LOG.md:259`).

So the honest prior is that this loses to Chapter 4. **That still discharges C3.** §5.4's claim is that these memberships are FIS antecedents; measuring how they predict is the test, and a negative result is a finding rather than a failure — it would point straight at boundary resolution as the thing to fix, which is a better Chapter 5 research question than the one currently open. Do not tune until it wins; report what it does.

## Acceptance

- A supervised, held-out score from a model whose antecedents are `block_membership` output, on at least bodyfat, at ten seeds.
- The Chapter 4 arm on identical splits, for comparison.
- If it loses, the write-up says so plainly and connects it to the `phase6` boundary-resolution finding.

**Comment by fundthmcalculus** [comment]
**Closed by `8deb499`. The result is negative, and sharper than this issue anticipated.**

The arrow is joined: `MS.block_membership` → `solve_tsk_consequents_from_firing` → `apply_tsk_consequents`, scored held out. (Not `predict_tsk` — it recomputes firing from a `GaussianMixtureModel` and cannot take a precomputed `U`.) Two venues, ten seeds, every arm on identical splits.

**bodyfat**, 10 × 5-fold:

| arm | held-out R² |
|---|---:|
| floor: OLS, all 13 features | **0.698 ± 0.005** |
| floor: OLS, the three used columns | **0.668 ± 0.007** |
| antecedent-free 1-rule TSK | 0.662 ± 0.010 |
| Ch5 crisp `MS.assign` → TSK | 0.659 ± 0.013 |
| Ch5 graded `U` → TSK | 0.653 ± 0.013 |
| Ch4 `TribbleRegressor` | 0.633 ± 0.036 |
| floor: train mean | −0.024 ± 0.015 |

Chapter 5 beats Chapter 4 — and that comparison is worthless on its own. The antecedent-free arm (same features, order, ridge; antecedents **deleted**) beats it, paired −0.0085 ± 0.0089 across 8/10 seeds, and every fuzzy arm loses to plain OLS.

**The mechanism turned out to be structural rather than a property of bodyfat.** Over all 50 folds, with zero variance: `max μ over non-members = 0.5000 ± 0.0000`. That is a theorem. A non-member reaches a single-linkage block only across the edge that dissolves it, so `d_B(x) ≥ death_B`, and `block_membership`'s Gaussian sits at half-max exactly at `death`. **`block_membership` can never place a non-member above 0.5, on any data.**

That upgrades `phase6_soft_validation` from an empirical finding to a structural one, and corrects its wording: *"a constant step per cluster"* is the two-block case — here `U` takes 81.7 distinct values across 252 points. The accurate statement is **no resolution above half-max**.

**ECG5000 carries the positive**, and it is the venue that matters more (DTW, coordinate-free): graded 0.897 ± 0.006, crisp 0.898 ± 0.005, inductive 0.905/0.909, against a 0.584 majority floor — with the antecedent-free control landing *exactly* on the floor. There the antecedents genuinely are informative, and it survives the leak-free inductive protocol.

Two corrections to the framing in this issue:
- **Graded vs crisp is protocol-dependent.** Transductively crisp leads; inductively graded leads (+0.0200 ± 0.0159, 9/10 seeds), and ECG5000's inductive pair goes the other way. Real, small, sign-unstable — not the clean loss I predicted.
- The prediction that it would "lose to Chapter 4" was wrong in letter. It beats Chapter 4 and loses to *having no antecedents at all*, which is a more useful finding.

Still owed before any of this is quoted: `data/bodyfat.csv` is untracked and absent from `reproduce/dataset_specs.yaml` and `PROVENANCE_MAP.md`.
